app/assets/stylesheets/scaffolds.scss

```diff
@@ -1,71 +0,0 @@
-/* Commenting all of this to give Bulma a clean slate
```

Comment:
Removing this scaffolded file, and changed scaffold settings so this won't get recreated